src/utils/platformText.ts

```diff
@@ -0,0 +1,30 @@
+import os from 'os';
+
+export function getCopyShortcut() {
+  if (os.platform() === 'darwin') {
+    return '⌘⇧C';
+  }
+
+  return 'Ctrl+Shift+c';
+}
+
+export function getOpenText() {
+  switch (os.platform()) {
+    case 'darwin':
+      return 'Open in finder';
+    case 'win32':
+      return 'Open in explorer';
+    case 'linux':
+      return 'Open in file manager';
+    default:
+      return 'Open in file manager';
+  }
+}
+
+export function getSearchShortcut() {
+  if (os.platform() === 'darwin') {
+    return '⌘f';
+  }
+
+  return 'Ctrl+f';
+}
```

Comment:
This would mean for every OS based text you have to create a new fn here.
A general util fn that just selects a selects a string from an object would be much better.

Sudo code:
```js
export function platformBasedText({
mac,win,linux
}: { 
mac:string;
win:string;
linux:string;
}) {
if (mac) {
 return mac;
} else if(win) { ... 
} else {
 return linux;
}

}
```

Reply:
Makes sense, have made the required change